src/collectives_c.c

```diff
@@ -392,9 +392,14 @@ shmem_longdouble_min_to_all(long double *target, long double *source, int nreduc
 {
     SHMEM_ERR_CHECK_INITIALIZED();
 
-    shmem_internal_op_to_all(target, source, nreduce, sizeof(long double),
+    if(!shmem_long_dub_supported) {
```

Comment:
in this case I'd have to undo some op.h work I've done. I've specialized everything for only the long double case. What are the combinations we are are anticipating? Should I just re-generalize for any atomic combination case?